e2e/teams.loggedin.spec.ts

```diff
@@ -60,27 +72,27 @@ test.describe('Teams', () => {
       ).toBeVisible()
     })
 
-    await test.step('user should invite admin to team', async () => {
+    await test.step('user should invite validated user to team', async () => {
       await userPage.getByRole('button', { name: 'Team Actions' }).click()
       await userPage.getByRole('link', { name: 'Invite User' }).click()
 
       await userPage
         .getByRole('textbox', { name: 'Enter a name or email address...' })
-        .type(adminUserName!)
+        .type(validatedUserName)
       await userPage
-        .getByRole('menuitem', { name: `@${adminUserName}` })
+        .getByRole('menuitem', { name: `@${validatedUserName}` })
         .click()
       const loadInvitedUser = userPage.getByText('Loading...')
       await expect(loadInvitedUser).toBeVisible()
       await expect(loadInvitedUser).not.toBeVisible()
 
-      await userPage.getByLabel('Invitation Message').type(INVITATION_MESSAGE)
+      const inviteMessageBox = userPage.getByLabel('Invitation Message')
+      await inviteMessageBox.click()
+      await inviteMessageBox.fill(INVITATION_MESSAGE)
+      await expect(inviteMessageBox).toHaveValue(INVITATION_MESSAGE)
```

Comment:
Ensure that the invitation text has been entered into the message box the invitation is sent, which prevents the test from failing in CI only

Reply:
Nice catch, input must be slow/flaky in CI